mayavi/core/lut_manager.py

```diff
@@ -22,9 +22,30 @@
 from mayavi.core.common import error
 
 from mayavi.core import lut
+
+# A logger for this module.
+logger = logging.getLogger(__name__)
+
+# Colormap from pickled file
 lut_image_dir = os.path.dirname(lut.__file__)
-pylab_luts = state_pickler.load_state(os.path.join(lut_image_dir,
-                                                'pylab_luts.pkl'))
+pylab_luts_file = os.path.join(lut_image_dir, 'pylab_luts.pkl')
+
+try:
+    pylab_luts = state_pickler.load_state(pylab_luts_file)
+except (IOError, ValueError) as exception:
+    # IOError: failed to open file
+    # ValueError: pickled file is built from an OS w/ different architecture,
+    #             or with an incompatible protocol
+    message = ("Failed to load pylab colormaps from file: {filepath}\n"
```

Comment:
I think we should dump os specific files and load them in that case instead of failing and asking the user to build this.